src/test/scala/com/cognite/spark/datasource/AssetsRelationTest.scala

```diff
@@ -58,7 +55,7 @@ class AssetsRelationTest extends FlatSpec with Matchers with SparkTest {
     assert(df.count() == 10)
   }
 
-  it should "support pushdown filters on name and depth" taggedAs ReadTest in {
+  it should "support pushdown filters on name and depth" taggedAs ReadTest ignore {
```

Comment:
Rather than ignoring it we should change it to not use depth, I guess?

Reply:
Just ignoring the tests for things I haven't implemented yet, such as pushdown filters and upsert.

Reply:
Alright then, let's fix that next. :)